tests/drivers/build_all/stepper/gpio.dtsi

```diff
@@ -21,3 +21,14 @@ test_uln2003_motor_cluster: uln2003_motor_cluster {
 			<&test_gpio 0 0>;
 	};
 };
+
```

Comment:
I think you need to rebase on main once? Gpio stepper does not support child nodes anymore.

Reply:
On main the overlay still uses the child nodes?
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/build_all/stepper/gpio.dtsi

Reply:
My bad. Will fix it today evening.